src/helpers/linkHelper.js

```diff
@@ -23,15 +23,30 @@ function ensureProtocol(link) {
   return linkWithProtocol;
 }
 
+/**
+ * Check if a given link is a web page.
+ *
+ * @param {string} linkWithProtocol
+ *
+ * @return true if linkWithProtocol is a web page
+ */
+function isWeb(linkWithProtocol) {
+  return linkWithProtocol.match(/^https?:/);
+}
+
 // https://facebook.github.io/react-native/docs/linking.html#opening-external-links
```

Comment:
I can check what ensureProtocol does on this link

Reply:
the whole `Linking` stuff is documented with different cases.
i can see, that the documentation has updated to `async/await`. we are still using `.then(...)`, which achieves the same.